libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -44,9 +55,12 @@ class CCAPPCOMMON_LIB_API ccApplicationBase : public QApplication
 	void setupPaths();
 		
 	const QString c_VersionStr;
-	
+	// QString
+    // url: https://doc.qt.io/qt-5/qstring.html
 	QString	m_ShaderPath;
 	QString	m_TranslationPath;
```

Comment:
missing doc?

Reply:
rewriting the type and putting a link to Qt's doc is not a really helpful comment.

QString is such an important part of Qt that people wanting to know more about QString will easily find information on the internet

Reply:
ok I will remove that.